views/includes/inputs/address/residential-address-input.html

```diff
@@ -104,6 +104,9 @@
     {
       value: 1,
       text: "Yes",
+      hint: {
```

Comment:
The `hint` text should be passed from `views/beneficial-owner-individual.html` to avoid conflicting with the `views/managing-officer.html` file.

Reply:
There is already an example for `last-name-input`

Reply:
There's another ticket for this: https://companieshouse.atlassian.net/browse/ROE-1627 - It will be the same text.

Reply:
Nope

Reply:
This has now been updated as suggested.